doc/pages/options.asciidoc

```diff
@@ -327,8 +327,10 @@ are exclusively available to built-in options.
     interface implementation. The NCurses UI support the following options:
 
         *ncurses_set_title*:::
-            if *yes* or *true*, the terminal emulator title will
-            be changed
+            if not set to *no* or *false*, the terminal emulator
+            title will be changed. if set to *yes* or *true*, the
+            default title line will be used, otherwise the value
+            of this option will be used as the title line
```

Comment:
When I read this, it wasn't obvious to me how to make different windows have different titles (which seemed an important use-case). How about:

```suggestion
            If set to *no* or *false*, the terminal emulator
            title will not be changed.
            
            If set to *yes* or *true*, each window's title will
            mirror the contents of the window's modeline
            (as set by the `modelinefmt` option).
            
            Otherwise, each window will use the value of this
            option at window scope as the title text, without
            further expansion.
```

Reply:
That's clearer!